examples/event_handling/pipong.py

```diff
@@ -1,7 +1,10 @@
-# A matplotlib based game of Pong illustrating one way to write interactive
-# animation which are easily ported to multiple backends
-# pipong.py was written by Paul Ivanov <http://pirsquared.org>
+"""
+====
+PONG
+====
 
+A matplotlib based game of Pong illustrating one way to write interactive animation which are easily ported to multiple backends pipong.py was written by Paul Ivanov <http://pirsquared.org>
```

Comment:
Here and below, wrap your lines at 79 characters (most editors should provide some option to help that).